aqua/Aerial/Portfile

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               xcode 1.0
+PortGroup               github 1.0
+
+github.setup            JohnCoates Aerial 2.0.5 v
+
+categories              aqua
+license                 MIT
+platforms               macosx
+maintainers             {danchr @danchr} openmaintainer
+
+description             Apple TV Aerial Screensaver for Mac
+
+long_description        Aerial is a Mac screensaver based on the new \
+                        Apple TV screensaver that displays the Aerial \
+                        movies Apple shot over New York, San \
+                        Francisco, Hawaii, China, etc.
+
+# uses git submodules, not included in archive
+fetch.type              git
+
+post-fetch {
+    # fetch the only required submodule; cloning Sparkle fails
+    system -W ${worksrcpath} "git submodule update --depth 1 --init Extern/OAuthSwift"
+}
+
+# although the binary works on 10.9 and later, it doesn't build on
+# anything earlier than 10.15
+if {${os.major} == 19} {
+    patchfiles-append  patch-tintconfiguration.diff
+} elseif {${os.major} < 19} {
+    known_fail         yes
+
+    pre-fetch {
+        ui_error "${name} ${version} cannot build on macOS 10.14 and earlier."
+        return -code error "incompatible macOS version"
+    }
+}
+
+# add missing internal dependency to simplify build
+# https://github.com/JohnCoates/Aerial/pull/1085
+patchfiles-append      patch-xcode-dependency.diff
+
+if {[vercmp $xcodeversion 12.0] < 0} {
+    patchfiles-append  patch-explicit-self.diff
+}
+
+# this port installs a symlink into "/Library/Screen Savers"
+destroot.violate_mtree  yes
+
+set screen_saver_dir    "/Library/Screen Savers"
+
+# this is horrible, but anything within ${prefix} causes permission
+# errors, even without trace mode; my best guess is that Xcode does
+# some internal sandboxing, see also:
+# https://trac.macports.org/ticket/57234
+set custom_build_dir    ${worksrcpath}/build
+
+xcode.build.settings-append \
+                        BUILD_DIR=${custom_build_dir}
```

Comment:
Dan/Anyone, we're seeing buildbot failures on 10.11 for `MenuMeters`, with the classic symptoms related to CoreSimulator, etc. However, it looks like this was fixed for base, via [PR 208 - sandbox: handle /opt/local on separate mountpoint](https://github.com/macports/macports-base/pull/208).

Would you folks expect this issue to still occur on our buildbot?

https://build.macports.org/builders/ports-10.11_x86_64-builder/builds/160675/steps/install-port/logs/stdio

Reply:
And if the fix for base isn't enough, would we want to try the same fix that was tried here (relative to setting `BUILD_DIR`, etc)?

CC: @Tatsh

Reply:
> it's actually a compile error

Right:

```
MenuMetersMenuExtraBase.m:34:8: error: unexpected '@' in program
    if(@available(macOS 10.14,*)){
       ^
```